ej2-react/chart/getting-started.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Getting started with React Chart component | Syncfusion
+title: Getting started with React Chart Component | Syncfusion
 description:  Checkout and learn about Getting started with React Chart component of Syncfusion Essential JS 2 and more details.
```

Comment:
can you change the description like as below 
<img width="673" alt="image" src="https://github.com/syncfusion-content/ej2-react-docs/assets/44014283/7b02c886-c76b-4463-826a-fe2d0698d850">

Reply:
changed